frontend/packages/console-shared/src/components/spotlight/__tests__/Spotlight.spec.tsx

```diff
@@ -8,7 +8,10 @@ describe('Spotlight', () => {
   type SpotlightProps = React.ComponentProps<typeof Spotlight>;
   let wrapper: ShallowWrapper<SpotlightProps>;
   const uiElementPos = { height: 100, width: 100, top: 100, left: 100 };
-  const uiElement = { getBoundingClientRect: jest.fn().mockReturnValue(uiElementPos) };
+  const uiElement = {
+    getBoundingClientRect: jest.fn().mockReturnValue(uiElementPos),
+    getAttribute: jest.fn().mockReturnValue(false),
```

Comment:
Please add a unit test for the new functionality

Reply:
Added tests for hidden check